backend/tests/test_recovery_scoring.py

```
@@ -57,12 +57,17 @@ def _sofi_position(**overrides) -> dict:
     return base
 
 
-def _sofi_paths(target_yield: float | None = 0.18, sizing_cap: float = 5000.0):
+def _sofi_paths(
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `_sofi_paths`

(ANN202)